test/test_discordmessageprocessor.ts

```diff
@@ -404,7 +399,18 @@ describe("DiscordMessageProcessor", () => {
                 name: "blah",
             };
             const reply = processor.InsertEmoji(content);
-            Chai.assert.equal(reply, "\x01blah\x011\x011234\x01");
+            Chai.assert.equal(reply, "\x01emoji\x01blah\x011\x011234\x01");
+        });
+    });
+    describe("InsertChannel", () => {
+        it("inserts channels to their post-parse flag", () => {
```

Comment:
do we want to test multiple channel mentions to ensure it doesn't get itself confused?

Reply:
good idea